src/store/TranscriptionsStore.js

```diff
@@ -545,9 +547,12 @@ const TranscriptionsStore = types.model('TranscriptionsStore', {
     return count;
   },
 
-  get lockedByDifferentUser () {
-    const login = getRoot(self).auth.user.login;
-    return self.current.locked_by && self.current.locked_by !== login
+  get lockedByCurrentUser () {
+    const login = getRoot(self).auth.user && getRoot(self).auth.user.login;
+    if (login && self.current) {
+      return self.current.locked_by && self.current.locked_by === login
```

Comment:
Minor comment, mostly to test the logic capabilities of my brain:

Isn't the existential check of `self.current.locked_by &&` unnecessary, technically? I assume the check is there to ensure `self.current.locked_by` isn't null, but you'll never get the unwanted true value of `self.current.locked_by (null) === login (null)` since `login` can never be null/undefined due to the earlier `login &&` check.

That said, don't change this line of code - in the context of our JS coding style, your code is more readable this way.

Reply:
Yep, that's exactly correct. It does seem a bit strange to have `self.current.locked_by && self.current.locked_by === login`, but it's there to guard against null cases.